website/src/css/imports/base.css

```diff
@@ -56,7 +56,7 @@ h3 > p {
 }
 
 p, ul {
-  font-size: $tiny;
+  font-size: 18px;
```

Comment:
Not too worried about breaking from variables here, current variable scheme isn't too comprehensive, and will likely be rewritten in a 2.0-ish overhaul.